include/libkernelflinger/log.h

```diff
@@ -38,9 +38,15 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE * efi_log_file;
+UINTN      KERNELFLINGER_LOG_OUTPUT;
```

Comment:
**Coding style violation**: All upper case are reserved for constant.  In addition the variable could be way more concise. What about `log_output` ?

Reply:
Already modified.